model/src/test/kotlin/JsonSchemaTest.kt

```diff
@@ -99,6 +99,19 @@ class JsonSchemaTest : StringSpec() {
 
             errors should beEmpty()
         }
+
+        "reference.yml validates successfully" {
+            val ortConfigurationSchema = JsonSchemaFactory
+                .builder(JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V6))
+                .objectMapper(mapper)
+                .build()
+                .getSchema(File("../integrations/schemas/ort-configuration-schema.json").toURI())
+            val referenceConfigFile = File("src/main/resources/reference.yml").toJsonNode()
+
+            val errors = ortConfigurationSchema.validate(referenceConfigFile)
```

Comment:
I would inline `errors`

Reply:
I try to maintain the `given, when, then` structure, which requires the execution to be split from the verification.
It also follows the style of all other tests in this file.